src/main/ionic/peers/src/app/app.module.ts

```diff
@@ -32,6 +35,7 @@ import { AppState } from "../states/app-state";
     MyApp,
     HomePage,
     ListPage,
```

Comment:
Can you delete the ListPage declarations as well as the actual files?

Reply:
Yeah